gdc_client/download/client.py

```diff
@@ -65,12 +65,19 @@ def download_annotations(self, file_id):
         annotation_list = ','.join(annotations)
 
         if annotations:
-            log.debug('Found {0} annotations for {1}.'.format(
-                len(annotations), file_id))
+            log.debug('Found {0} annotations for {1}.'.format(len(annotations), file_id))
+            # TODO: Refactor this with the _post method for a generic function for calling active/legacy api
             r = requests.get(
                 urlparse.urljoin(self.data_uri, annotation_list),
                 params={'compress': True},
                 verify=self.verify)
+            if r.status_code != requests.codes.ok:
```

Comment:
not sure how feasible it's here, but we added a `203` response code for files that have newer versions. Maybe stick the check here as well, just in case.